json-spec/extensions/stac-collection-spec.md

```diff
@@ -0,0 +1,154 @@
+# STAC Collection Extension Spec
+
+A group of STAC `Item` objects from a single source usually shared common metadata. This is especially true with satellite imagery that uses the STAC EO Extension. Rather than including these common metadata fields on every `Item`, they can be grouped into a `Collection`.
+
+## Collection Extension Description
+
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| collection_name | string | Collection Name | A name given to the Collection
+| collection_description | string (optional) | Collection Description | A human readable description of the collection
+
+A `Collection` does not have many specific fields, as it may contain any fields that are in the core spec as well as any other extension. If a field is specified in `Collection`, it will be ignored in any `Item` that links to that `Collection`. This is important because a `Collection` is the metadta that is common across all `Item` objects. If a field is variable at all, it should be part of the `Item` record.
```

Comment:
What's the advantage of the collection field overriding the Item?

I general it seems to me better that if someone only looked at a single piece of data and didn't follow the collection link that they could count on a field being correct, as opposed to having to check the collection to see if it got overridden. 

 I was also thinking it could be a valuable feature to have Item's be able to override collections. Like if there's 'field_z' that 98% of items have a value of 'true' and 2% have 'false' then an implementer could just stick field_z='true' at the collection level.

But I'm probably just missing something as to why that route would be bad.

Reply:
I was originally thinking that it could be default values, but that could be overridden.  
The problem I see with that is the main reason Collection level metadata is useful to the developer is because you *know* they are all the same.  If my bands are described in the collection level metadata than I can use that for all my items. If that data isn't common across all items then I'm back to having to check both, and then somehow reconcile them if they are different.  IMHO it ends up defeating the whole purpose for having collection level metadata to begin with. It's most important with the band information, I really need to know that the all of my items contain the same band metadata, but I could see it as needed for other fields as well.

OTOH I do see overriding as valuable too, but it's really important to know whether or not the field really is in 100% of the items or it is not.  I could see this as a feature of collections, being able to specify if a field is overridable or not.